.github/workflows/e2e-fixture-test.yml

```diff
@@ -39,3 +39,9 @@ jobs:
         USE_EXISTING_CLUSTER: "false"
         GINKGO_NODES: 1
       run: make test-e2e
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+      with:
+        name: artifacts.tar.gz
+        path: test/e2e/_artifacts
```

Comment:
Do we want to explicitly specify options `if-no-files-found` or `overwrite` ? They default to `warn` and `fail`.

Reply:
I don't have a strong opinion on that. What would you prefer?

Reply:
Good point! I have made them explicit now and also changed it so that it will fail if no files are found.